docs/source/metrics.rst

```diff
@@ -17,6 +17,34 @@ value is then computed using the output of the engine's ``process_function``:
     metric = Accuracy()
     metric.attach(engine, "accuracy")
 
+If the engine's prediction output ``y_pred`` represents probability estimates, it can be binarized using the
+``Mode.PROBABILITIES``:
```

Comment:
Maybe, it would worth to say which threshold is used by default to transform probas to binary predictions.